.github/workflows/templates.yml

```diff
@@ -53,6 +23,6 @@ jobs:
         SECRET_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         CACHIX_TOKEN: ${{ secrets.CACHIX_TOKEN }}
 
-    - name: Test devshell ${{ matrix.template }}
+    - name: Test devshells
       run: |
-        nix run .#tests.templates.${{ matrix.template }}
+        nix develop --accept-flake-config --command bash test-template.sh
```

Comment:
I wonder how long it will take without parallelisation...

Reply:
![image](https://github.com/nickel-lang/nickel-nix/assets/7226587/e7af3768-bf60-4c5a-b27f-463f314497ea)

Reply:
That's running BashShell 14 times ;)

Reply:
It's indeed a bit slower now :)

![image](https://github.com/nickel-lang/nickel-nix/assets/7226587/40018214-3059-4e2b-af56-6986e29b142f)

Do you think that justifies restoring the parallelism?

Reply:
After that Cachix pushed 116 paths. Maybe that's just cold cache? I've retriggered it here: https://github.com/nickel-lang/nickel-nix/actions/runs/5465840641/jobs/9950269605

Reply:
(Or should we just parallelize within a single runner? That might be even faster)

Reply:
I wonder how hard it would be. Just running multiple Bash jobs would make it impossible to read logs afterwards.
Maybe enabling recursive-nix is not that hard?.. :)

Reply:
By the way, it looks like CI is not strictly repeatable: we're using latest `nixpkgs` input on each run, so cache is probably getting invalidated more often than it should be. Maybe we should add `--inputs-from ${./.}` to `nix flake lock` there?

Reply:
> we're using latest nixpkgs input on each run, so cache is probably getting invalidated more often than it should be

Duh, indeed :grimacing: